apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/ServerTestUtils.java

```diff
@@ -399,6 +405,103 @@ public static List<Object> parseData(List<StaticRifResource> sampleResources) {
     return recordsParsed;
   }
 
+  /**
```

Comment:
Moved these methods out of the EOB restAssured test, since all the new RestAssured tests needed them too.